ocs_ci/ocs/resources/pod.py

```diff
@@ -592,8 +601,25 @@ def get_pod_obj(name, namespace=None):
 
     Returns:
         obj : A pod object
+
     """
     ocp_obj = OCP(api_version='v1', kind=constants.POD, namespace=namespace)
     ocp_dict = ocp_obj.get(resource_name=name)
     pod_obj = Pod(**ocp_dict)
     return pod_obj
+
+
+def delete_pods(pod_objs):
```

Comment:
This function assumes "namespace". pls, fetch namespace as an argument for this function.